docs/database/elasticsearch.md

```diff
@@ -86,5 +94,7 @@ the server address behind any intermediaries (e.g. proxies) if it's available.
 | `db.operation`                      | `"search"`                                                                                                                          |
 | `url.full`                          | `"https://elasticsearch.mydomain.com:9200/my-index-000001/_search?from=40&size=20"`                                                 |
 | `db.elasticsearch.path_parts.index` | `"my-index-000001"`                                                                                                                 |
+| `db.elasticsearch.cluster_name`     | `"e9106fc68e3044f0b1475b04bf4ffd5f"`                                                                                                |
```

Comment:
Need to adapt the examples here, now that's using a namespace.

Reply:
Of course!